about/people/developers.md

```diff
@@ -0,0 +1,28 @@
+---
+title: 'Volunteer Developers'
+---
+
+# Volunteer Developers
+
+The technical infrastructure and software behind arXiv.org is operated and
+maintained by a core team of dedicated IT staff at Cornell. However, it would
+not be possible to keep pace with the ever-changing landscape of internet
+technology, nor the evolving needs of 21st century science, without the help of
+dedicated arXiv users who contribute to the open source software that powers
+the arXiv platform.
+
+We are grateful to the following external developers for their contributions to
```

Comment:
Do you want to use "volunteer" in place of "external" developers?

Reply:
Yes, good idea; fixed in b7880fe